packages/table/test/tableTests.tsx

```diff
@@ -622,15 +623,15 @@ describe("<Table>", function(this) {
 
             const table2 = mount(<Table numFrozenColumns={1} />);
             expect(table2.state("numFrozenColumnsClamped")).to.equal(0);
-            expect(consoleWarn.callCount).to.equal(1, "warned 1");
+            expect(consoleWarn.callCount).to.equal(2, "warned 1");
```

Comment:
these callCount changes throughout this suite are a little concerning to me; why do our expectations about the table behavior have to change in this PR?

Reply:
this is not done. i plan on removing the `validateProps` call in the constructor that leads to the double warnings